contrib/chatops/README.jinja

```diff
@@ -0,0 +1,68 @@
+# {{ pack["pack.yaml"].name }} integration pack v{{ pack["pack.yaml"].version }}
```

Comment:
curious how you used this to generate the readme?

Reply:
I used pack2md https://github.com/nzlosh/pack2md.  There's a footnote in the README.md